src/script_chainer/gui/page/script_setting_interface.py

```
@@ -378,30 +462,156 @@ def on_edit_clicked(self) -> None:
+        self.attach_changed.emit()
+
+    def on_run_clicked(self) -> None:
+        """运行 Python 脚本"""
+        path = self.config.script_path
+        if not path or not os.path.exists(path):
+            _show_warning(self.window(), '无法运行', 'Python 脚本文件不存在')
+            return
+        if getattr(sys, 'frozen', False):
+            python = shutil.which('python') or shutil.which('python3')
+            if not python:
+                _show_warning(self.window(), '无法调试', '未找到系统 Python，请安装 Python 并添加到 PATH')
```

Ruff (0.15.10)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)